packages/dashboard-frontend/src/services/bootstrap/__tests__/index.spec.tsx

```diff
@@ -92,11 +93,6 @@ describe('Dashboard bootstrap', () => {
     });
     expect(mockGet).toHaveBeenCalledWith('/dashboard/api/server-config', undefined);
     expect(mockGet).toHaveBeenCalledWith('/dashboard/api/cluster-info');
-    expect(mockGet).toHaveBeenCalledWith('/dashboard/api/userprofile/test-che', undefined);
```

Comment:
The line with `/dashboard/api/userprofile/test-che` should not be removed.

Reply:
Let's add one more:

```
expect(mockGet).toHaveBeenCalledWith('/dashboard/api/editors);
```

Reply:
done